curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md

```diff
@@ -25,18 +29,22 @@ const alpha = {
   26:"A"
 };
 
-const thirdLetter = alpha[2];
-const lastLetter = alpha[24];
+const thirdLetter = reverseAlphabet[2];
+const lastLetter = reverseAlphabet[25];
```

Comment:
The change from `alpha` to `reverseAlphabet` is the only part of this PR that I think we should keep, honestly